nvflare/app_common/tracking/log_writer_me.py

```diff
@@ -40,7 +40,7 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
             if self.sender is None:
                 self.task_panic("Cannot load MetricsExchanger!", fl_ctx=fl_ctx)
 
-    def log(self, key: str, value: Any, data_type: AnalyticsDataType, **kwargs):
+    def send_log(self, key: str, value: Any, data_type: AnalyticsDataType, **kwargs):
```

Comment:
I am good with this name.
@chesterxgchen @holgerroth please double check if this name clear enough for you thanks.